apps/web/src/components/user-form.tsx

```diff
@@ -140,6 +146,25 @@ export const UserForm = ({ user, degrees }: UserFormProps) => {
           )}
         />
 
+        <Link href={retningslinjer} >Retnlingslinjene</Link>
+
+          <form onSubmit={form.handleSubmit(onSubmit)} className="space-y-6">
+            <FormField
+              control={form.control}
+              name="hasReadTerms"
+              render={({ field }) => (
+                <FormItem className="flex flex-row items-start space-x-3 space-y-0">
+                  <FormControl>
+                    <Checkbox checked={field.value} onCheckedChange={field.onChange} />
+                  </FormControl>
+                  <div className="space-y-1 leading-none">
+                    <FormLabel>Jeg bekrefter at jeg har lest retningslinjene.</FormLabel>
```

Comment:
Jeg hadde flyttet linken til retningslinjene her.

```suggestion
                    <FormLabel>Jeg bekrefter at jeg har lest <Link href="/retningslinjer.pdf" className="underline font-semibold">retningslinjene</Link>.</FormLabel>
```